src/psyclone/psyGen.py

```diff
@@ -74,6 +86,17 @@
 # List of all valid access types for a kernel argument
 VALID_ACCESS_DESCRIPTOR_NAMES = []
 
+# Colour map to use when writing Invoke schedule to terminal. (Requires
+# that the termcolor package be installed. If it isn't then output is not
+# coloured.) See https://pypi.python.org/pypi/termcolor for details.
+SCHEDULE_COLOUR_MAP = {"Schedule": "yellow",
+                       "Loop": "blue",
+                       "GlobalSum": "cyan",
+                       "Directive": "green",
+                       "HaloExchange": "magenta",
+                       "Call": "red",
```

Comment:
Should BuiltIns have a different colour to Kernel calls?

Reply:
Well, they could :-) I've taken this to mean you'd like them to and since I am agnostic I've changed the colour map. There are only two other colors (grey and white) available and grey is pretty much invisible on my system. I'm therefore using red and magenta for built-ins and Kernels, blue and cyan for halo swaps and global sums and white for loops.